test/CrateTest/PDO/PDOTest.php

```diff
@@ -125,8 +133,8 @@ public function testSetAttributeWithHttpBasicAuth()
             ->method('setHttpBasicAuth')
             ->with($user, $passwd);
 
-        $pdo->setAttribute(PDO::ATTR_HTTP_BASIC_AUTH, [$user, $passwd]);
-        $this->assertEquals($expectedCredentials, $pdo->getAttribute(PDO::ATTR_HTTP_BASIC_AUTH));
+        $pdo->setAttribute(PDO::CRATE_ATTR_HTTP_BASIC_AUTH, [$user, $passwd]);
```

Comment:
do you also want to write a similar test for the CRATE_ATTR_DEFAULT_SCHEMA attribute?